spec/models/page_content_register_spec.rb

```diff
@@ -30,9 +30,21 @@
 
     context 'when page is the homepage' do
       let(:layout) { double(identifier: 'home_page') }
+      let(:new_blocks_attributes) do
+        [{ content: 'super awesome content' },
+         { identifier: 'raw_tile_1_image', content: 'http://e.co/original/img.png' }]
+      end
 
-      it 'returns processed content as blank' do
-        expect(subject).to eq(new_blocks_attributes)
+      context 'image tile srcset' do
+        it 'is added to blocks attributes' do
+          blocks_identifiers = subject.map { |b| b[:identifier] }
+          expect(blocks_identifiers).to include('raw_tile_1_srcset')
+        end
```

Comment:
I would change the context "when identifier is 'raw_title_\d_image'" and on the it I would change to " replaces identifier 'image'  with 'srcset'. Do you think is it more clear this way?